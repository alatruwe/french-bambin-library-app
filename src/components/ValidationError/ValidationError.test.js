import React from "react";
import ReactDOM from "react-dom";
import ValidationError from "./ValidationError";
import renderer from "react-test-renderer";

describe(`ValidationError component`, () => {
  it("renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(<ValidationError />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it("renders the UI as expected", () => {
    const tree = renderer.create(<ValidationError />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
