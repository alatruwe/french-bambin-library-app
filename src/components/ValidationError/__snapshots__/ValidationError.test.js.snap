// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ValidationError component renders the UI as expected 1`] = `null`;
